cypress/integration/main_page_spec.js

```diff
@@ -60,26 +60,53 @@ describe('Dashboard Home Page', () => {
     });
 
     it('displays a compatible Cumulus API Version number', () => {
-      const apiVersionNumber = 'a.b.c';
+      const apiVersionNumber = 'a.b.c-d.e';
```

Comment:
Did we change the way the version number is displayed?

Reply:
Yes, it was changed in a previous ticket.

Reply:
Is that previous ticket a part of this PR? I guess my question is: why is the change here?

Reply:
The previous ticket is not part of this PR. The test kept failing at this step. My commit even states it was to fix test errors.

Reply:
Hmm so... what happened? The test was failing on main?

Reply:
When I ran cypress main_page_spec.js was failing and it was because of apiVersionNumber had a dash in it.